packages/shared/lib/core/network/constants/network-metadata-map.constant.ts

```diff
@@ -1,9 +1,10 @@
 import { COIN_TYPE, DEFAULT_BASE_TOKEN } from '../constants'
 import { NetworkId, TokenSupply } from '../enums'
-import { INetwork } from '../interfaces'
+import { IStardustNetworkMetadata } from '../interfaces'
+import { NetworkMetadataMap } from '../types'
 
-export const NETWORK: Readonly<{ [key in NetworkId]?: INetwork }> = {
-    [NetworkId.Iota]: {
+export const NETWORK_METADATA_MAP: Readonly<NetworkMetadataMap> = {
+    [NetworkId.Iota]: <IStardustNetworkMetadata>{
```

Comment:
This has chains? but is using IStardustNetworkMetadata which does not. Should the NETWORK_METADATA_MAP not be DEFAULT_PERSISTED_NETWORKS and not obfiscate the type from this file because then we would have seen this issue?